lib/cocoapods/command/outdated.rb

```diff
@@ -90,7 +93,7 @@ def deprecated_pods
 
       def spec_sets
         @spec_sets ||= begin
-          analyzer.send(:update_repositories) unless config.skip_repo_update?
+          analyzer.send(:update_repositories) if repo_update?(:default => true)
```

Comment:
I agree with @orta here. If you use it manually, you want probably always make sure that you're up-to-date.